tests/build/chains.go

```diff
@@ -13,35 +12,36 @@ import (
 
 var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", func() {
 	defer g.GinkgoRecover()
-	commonController, err := common.NewSuiteController()
-	Expect(err).NotTo(HaveOccurred())
-	tektonController, err := tekton.NewSuiteController()
+
+	// Initialize the tests controllers
+	framework, err := framework.NewFramweork()
 	Expect(err).NotTo(HaveOccurred())
+
 	ns := "tekton-chains"
```

Comment:
Can we Maintain the namespace strings in constants.go and just use them here ?
example: `ns := constants.TEKTON_CHAINS_NS`

Reply:
we don't need to declare a new variable:

just use: 
```go 
constants.TEKTON_CHAINS_NS
``` where is used now ```ns```